### PR TITLE
container for architect

### PR DESCRIPTION
**Pull request by TheoBrigitte**
Towards: https://github.com/giantswarm/giantswarm/issues/6025

Create a docker image for architect to be used from circleci.
This docker image contains architect + other tools we need.

Tools required from our side:
* architect
* git
* helm

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+FROM alpine:3.9
+
+ARG HELM_VERSION=v2.14.0
+
+RUN apk add --no-cache \
+        ca-certificates \
```
Comment (JosephSalisbury) [inline Dockerfile]: Is it really nitpicky to ask towards having specific versions pinned in the Dockerfile?

Comment (TheoBrigitte) [reply Dockerfile]: I think this is legit.

Comment (JosephSalisbury) [reply Dockerfile]: Okay cool, then let’s do that :D I’m bored of dependencies changing underneath us

Comment (TheoBrigitte) [reply Dockerfile]: done. PTALA

**Pull request merged**